BAZProject/MovieDetail/MovieDetailViewController.swift

```diff
@@ -10,12 +10,32 @@
 
 import UIKit
 
-class MovieDetailViewController: UIViewController, MovieDetailViewProtocol {
+final class MovieDetailViewController: UIViewController, MovieDetailView {
 
-	var presenter: MovieDetailPresenterProtocol?
+	var presenter: MovieDetailPresenter?
+    var movie: MovieModel?
 
-	override func viewDidLoad() {
+    @IBOutlet weak var image: UIImageView!
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblReleaseDate: UILabel!
+    @IBOutlet weak var lblVotes: UILabel!
+    @IBOutlet weak var lblOverview: UITextView!
+
+    override func viewDidLoad() {
         super.viewDidLoad()
+        configView()
+    }
+
+    func configView() {
+        guard let movie = movie else {
+            return
+        }
+        lblTitle.text = movie.movie.title
+        lblReleaseDate.text = "Fecha de estreno: \(movie.movie.releaseDate ?? "NA")"
+        lblVotes.text = "Votos: \(String(movie.movie.voteAverage ?? 0.0))"
+        lblOverview.text = movie.movie.overview
+        image.layer.cornerRadius = 10.0
+        image.image = movie.movieBackdrop
```

Comment:
El concatenar strings también podría ser considerado lógica de formateo, algo que el view controller no debería estar haciendo.
Todo el formato de textos, etc. debería ser responsabilida del presentador. El presentador debe proveer la vista los datos tal cual como la vista los necesita. Deberíamos mover este código al presentador.
```suggestion 
        lblTitle.text = movie.movie.title
        lblReleaseDate.text = "Fecha de estreno: \(movie.movie.releaseDate ?? "NA")"
        lblVotes.text = "Votos: \(String(movie.movie.voteAverage ?? 0.0))"
        lblOverview.text = movie.movie.overview
        image.layer.cornerRadius = 10.0
        image.image = movie.movieBackdrop
```